charts/sonarqube-dce/README.md

```diff
@@ -72,6 +72,16 @@ $ helm delete kindly-newt
 
 Please read the official documentation prerequisites [here](https://docs.sonarqube.org/latest/requirements/prerequisites-and-overview/).
 
+### Kubernetes - Pod Security Standards
+
+Pod Security Standards
+
+The following Pod Security levels cannot be used in combination with SonarQube's chart:
```

Comment:
Can you please add the link to those levels? https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels